index.html

```diff
@@ -83,9 +83,11 @@
   <body>
     <section id="abstract">
       <p>
-This specification defines how Verifiable Credentials can be represented as JSON
-Web Tokens (JWT) [RFC7519] and secured using JSON Web Signatures (JWS)
-[RFC7515].
+This specification defines how to represent Verifiable Credentials as JSON
+Web Tokens (JWT) [RFC7519] and secure them using JSON Web Signatures (JWS)
```

Comment:
I am not sure "represent as a JWT" is accurate. JWS is used to sign JSON-LD payload, which is not exactly a JWT.

Reply:
Hmm. Maybe this (to replace lines 86&87 above, not just the line 87 to which @Sakurann's comment is tied)?
```
This specification defines how to represent Verifiable Credentials as JSON
Web Tokens (JWT) [RFC7519]. It also defines how to represent Verifiable 
Credentials as JSON-LD, and how to secure such JSON-LD payloads using JSON 
Web Signatures (JWS)
```

Reply:
@Sakurann a JWT has all optional fields, so technically any JWS payload is a JWT payload, since all the fields are optional...

```
   The set of claims that a JWT must contain to be considered valid is
   context dependent and is outside the scope of this specification.
   Specific applications of JWTs will require implementations to
   understand and process some claims in particular ways.  However, in
   the absence of such requirements, all claims that are not understood
   by implementations MUST be ignored.
```

https://www.rfc-editor.org/rfc/rfc7519#section-4

But I agree, the language is not great here... we should say:

```
This specification defines how to secure Verifiable Credentials with JSON
Web Tokens (JWT) [RFC7519], which relies on JSON Web Signatures (JWS)...
```

Reply:
Small tweak --
```
This specification defines how to secure Verifiable Credentials with JSON
Web Tokens (JWT) [RFC7519], which rely on JSON Web Signatures (JWS)...
```